src/app/modules/customer-management/components/projects/components/services/project.service.ts

```diff
@@ -21,12 +22,16 @@ export class ProjectService {
     return this.http.post<Project[]>(this.url, {
       ...projectData,
       customer_id: 'b6e6a2f1-ce5c-49b3-8649-d154b4f3c305',
-      tenant_id: 'b95385bd-d0f4-4823-b874-af9421c14c8e',
+      tenant_id: this.azureAdB2CService.getTenantId(),
```

Comment:
we no longer need a tenant_id, this value is going to be read from the JWT so feel free to remove it 😃